src/main/webapp/item-page.html

```diff
@@ -1,37 +1,33 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8" />
-    <title>Item Page</title>
-    <!-- Bootstrap CSS -->
-    <link
-      rel="stylesheet"
-      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"
-    />
-    <link rel="stylesheet" href="style.css" />
-    <script src="itempagescript.js"></script>
-    <!-- Load an icon library -->
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
-    <!-- Javascript for Bootstrap -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
-    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
-  </head>
 
-  <body onload="loadItemPage()">
-    <div class="container">
-      <div id="item-container"></div>
-      <footer>
-        <ul id="comment-container"></ul>
-        <form>
-          <textarea name="comment"></textarea>
-          <br /><br />
-          <button onclick="sendFormData()">Submit</button>
-        </form>
-      </footer>
-    </div>
-  </body>
-</html>
+<head>
```

Comment:
remove this file from your PR? we can keep the old formatting

Reply:
Would deleting this file mess anything up?

Reply:
If you delete it and push the changes then this PR might end up deleting Eric's page. I think it's better just to restore it to how it looks in master and then push again so that this change doesn't show up in the PR.

Reply:
I copied and pasted the code from master.